File: 03. Ekspresi dan Masukan atau Keluaran/A tambah B/A-tambah-B.c
// memasukkan file header stdio.h
#include <stdio.h>

// membuat main function (fungsi utama) program
int main ()
{
    // deklarasi (menentukan tipe data dan penamaan variabel) dan inisialisasi (memberikan nilai variabel) variabel A dan B
    unsigned short A, B = 0;

    // input nilai variabel A dan B dari layar
    scanf ("%hu %hu", &A, &B);
    
    // mencetak output ke layar dengan menjumlahkan nilai variabel A dan B
    printf ("%hu", A + B);
    
    // mengakhiri program dan mengembalikan nilai 0 ke sistem operasi (nilai 0 artinya program berjalan dengan sukses)
    return 0;
}

File: 03. Ekspresi dan Masukan atau Keluaran/A tambah B/AtambahB.java
// mengimpor class Scanner dari library java.util
import java.util.Scanner;

// deklarasi class bernama AtambahB
public class AtambahB
{
    // titik awal eksekusi
    public static void main (String [] args)
    {
        // membuat objek Scanner bernama scanner untuk membaca input
        Scanner scanner = new Scanner (System.in);

        // input nilai variabel A dan B dari layar
        short A = scanner.nextShort ();
        short B = scanner.nextShort ();

        // Menutup objek Scanner setelah selesai membaca input
        scanner.close ();
        
        // mencetak output ke layar dengan menjumlahkan nilai variabel A dan B
        System.out.println (A + B);
    }
}